### PR TITLE
chore: Rename Makefile targets to upcgm

### PR DESCRIPTION
**Pull request by Yag000**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,7 +10,7 @@ help:
 	@echo ""
 	@echo "Available commands are:"
 	@echo "	make fast		- to build KACTL, quickly (only runs LaTeX once)"
-	@echo "	make kactl		- to build KACTL"
+	@echo "	make $(PROJECT_NAME)              - to build KACTL"
 	@echo "	make clean		- to clean up the build process"
 	@echo "	make veryclean		- to clean up and remove kactl.pdf"
```
Comment (YanisLcn) [inline Makefile]:
```suggestion
	@echo "	make $(PROJECT_NAME)              - to build UPCGM"
	@echo "	make clean		- to clean up the build process"
	@echo "	make veryclean		- to clean up and remove upcgm.pdf"
```

File: Makefile [hunk 2]
```diff
@@ -7,7 +10,7 @@ help:
 	@echo ""
 	@echo "Available commands are:"
 	@echo "	make fast		- to build KACTL, quickly (only runs LaTeX once)"
```
Comment (YanisLcn) [inline Makefile]:
```suggestion
	@echo "	make fast		- to build UPCGM, quickly (only runs LaTeX once)"
```

File: README.md [hunk 3]
```diff
@@ -1,9 +1,10 @@
-# KACTL
+# UPCGM Notebook
 
-This repo hosts KACTL, [KTH](https://en.wikipedia.org/wiki/KTH_Royal_Institute_of_Technology)'s ICPC team reference document.
-It consists of 25 pages of copy-pasteable C++ code, for use in ICPC-style programming competitions.
+This repo hosts UPCGM, [UPC campus Grands m=moulins](https://en.wikipedia.org/wiki/Universit%C3%A9_Paris_Cit%C3%A9)'s ICPC team reference document.
```
Comment (YanisLcn) [inline README.md]:
```suggestion
This repo hosts UPCGM, [UPC campus Grands Moulins](https://en.wikipedia.org/wiki/Universit%C3%A9_Paris_Cit%C3%A9)'s ICPC team reference document.
```

**Review by YanisLcn — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (Yag000) [reply Makefile]: I'd rather use the constants `PROJECT_NAME` and PDF. Changing that asap

**Review by YanisLcn — APPROVED** [review]

**Review by GabinDDL — APPROVED** [review]

**Pull request merged**